string message # a message describing the current task
float32 completion # a percentage (on [0,1]) describing the total progress
